examples/plot_stack_masks.py

```diff
@@ -0,0 +1,131 @@
+# the locations of cloudy and shadowed pixels in the scene.
+
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+
+# Stack the landsat bands
+# This creates a numpy array with each "layer" representing a single band
+landsat_paths_pre = glob(
+    "data/cold-springs-fire/landsat_collect/LC080340322016070701T1-SC20180214145604/crop/*band*.tif"
+)
+landsat_paths_pre.sort()
+arr_st, meta = es.stack(landsat_paths_pre)
+
+# Import the landsat qa layer
+with rio.open(
+    "data/cold-springs-fire/landsat_collect/LC080340322016070701T1-SC20180214145604/crop/LC08_L1TP_034032_20160707_20170221_01_T1_pixel_qa_crop.tif"
+) as landsat_pre_cl:
+    landsat_qa = landsat_pre_cl.read(1)
+    landsat_ext = plotting_extent(landsat_pre_cl)
+
+###############################################################################
+# Plot Histogram of Each Band in Your Data
+# ----------------------------------------
+
+# You can view a histogram for each band in your dataset by using the
+# ``hist()`` function from the ``earthpy.plot`` module.
+
+ep.hist(arr_st)
+plt.show()
+
+###############################################################################
+# Customize Histogram Plot with Titles and Colors
+# -----------------------------------------------
+
+ep.hist(
+    arr_st,
+    colors=["blue"],
+    title=[
+        "Band 1",
+        "Band 2",
+        "Band 3",
+        "Band 4",
+        "Band 5",
+        "Band 6",
+        "Band 7",
+    ],
+)
+plt.show()
+
+###############################################################################
+# View Single Band Plots
+# -----------------------------------------------
+# Next, have a look at the data, it looks like there is a large cloud that you
+# may want to mask out.
+
+ep.plot_bands(arr_st)
+plt.show()
+
+
+###############################################################################
+# Mask the Data
+# -----------------------------------------------
+
+# You can use the earthpy ``mask()`` function to handle this cloud.
+# To begin you need to have a layer that defines the pixels that
+# you wish to mask. In this case, the ``landsat_qa`` layer will be used.
+
+ep.plot_bands(
+    landsat_qa,
+    title="The Landsat QA Layer Comes with Landsat Data\n It can be used to remove clouds and shadows",
+)
+plt.show()
+
+
+###############################################################################
+# Plot The Masked Data
+# ~~~~~~~~~~~~~~~~~~~~~
+# Now apply the mask and plot the masked data. The mask applies to every band in your data.
+
+# Generate array of all possible cloud / shadow values
+cloud_shadow = [328, 392, 840, 904, 1350]
+cloud = [352, 368, 416, 432, 480, 864, 880, 928, 944, 992]
+high_confidence_cloud = [480, 992]
+
+# Mask the data
+all_masked_values = cloud_shadow + cloud + high_confidence_cloud
+arr_ma = em.mask_pixels(arr_st, landsat_qa, vals=all_masked_values)
+
+# sphinx_gallery_thumbnail_number = 5
+ep.plot_rgb(
+    arr_ma, rgb=[4, 3, 2], title="Array with Clouds and Shadows Masked"
```

Comment:
Where does this `[4, 3, 2]` come from?

Reply:
it's just whatever bands the user wants to plot. depending upon the data they will select different bands. this example uses Landsat 8.

Reply:
It would be useful to briefly describe where this array comes from in the vignette too, in case people aren't familiar with Landsat.

Reply:
I have added a sentence at the top of the vignette to introduce the data. hopefully that should clarify things!!